modules/distribution/src/repository/resources/conf/infer.json

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
Remove separation new lines. JSON we don't keep new lines.

Reply:
refactored in master

Reply:
Ack.